examples/getfile_putfile_config.json

```diff
@@ -0,0 +1,50 @@
+{
+    "rootGroup": {
+        "name": "MiNiFi Flow",
+        "processors": [
+            {
+                "name": "GetFile",
+                "identifier": "b01793b6-5275-4ee3-9fc0-d9e39ce3162f",
+                "type": "org.apache.nifi.processors.standard.GetFile",
+                "schedulingStrategy": "TIMER_DRIVEN",
+                "schedulingPeriod": "2 sec",
+                "properties": {
+                    "Input Directory": "/tmp/input"
```

Comment:
this seems to be different from its yaml counterpart, multiple properties seem to be missing for both processors

Reply:
Good catch, fixed in 20feddbeb43868d968be1a932939a2539c653eb8